### PR TITLE
5361 basecamp components

### PR DESCRIPTION
**Pull request by alysonturing**
Added new basecamp components

### CODE REVIEW

File: components/basecamp/basecamp.app.mjs [hunk 1]
```diff
@@ -1,11 +1,342 @@
...
+      },
+    },
+  },
   methods: {
-    // this.$auth contains connected account data
-    authKeys() {
-      console.log(Object.keys(this.$auth));
+    async getAccounts({ $ = this }) {
+      const { accounts } = await axios($, {
+        url: "https://launchpad.37signals.com/authorization.json",
```
Comment (feyzullah) [inline components/basecamp/basecamp.app.mjs:210]: Is the url correct? Just asking to make sure

**Review by feyzullah — COMMENTED** [review]
Hi @alysonturing, I left some comments for some minors. Can you check please?

File: components/basecamp/actions/create-comment/create-comment.mjs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+          projectId,
+          recordingType,
+        }) => ({
+          accountId,
+          projectId,
+          recordingType,
+        }),
+      ],
+    },
+    content: {
```
[the remark this answers is not in the document]
Comment (alysonturing) [reply components/basecamp/actions/create-comment/create-comment.mjs:47]: there's a lot of use for `content` prop but they all means different things, I could move it to app file but I would still have to change the description in every one of them and some places it is optional also

File: components/basecamp/basecamp.app.mjs (hunk 1 above)
Comment (alysonturing) [reply components/basecamp/basecamp.app.mjs:210]: yes, this url returns the accounts for the oAuth session, so we can use the accountId to the rest of the requests...

you can see more [here](https://launchpad.37signals.com/authorization.json)

**Review by feyzullah — APPROVED** [review]

**Comment by vunguyenhung** [comment]
Hi everyone, all test cases are passed! Ready for release! 

Test report
[Basecamp_5361.pdf](https://github.com/PipedreamHQ/pipedream/files/10734939/Basecamp_5361.pdf)

**Pull request merged**